docs/installation/providers/ubuntu.md

```diff
@@ -97,8 +97,9 @@ You should check out a tagged version of Monica since `master` branch may not al
 
 ```sh
 cd /var/www/monica
-git checkout tags/v2.2.1
+git checkout tags/v2.19.1
 ```
+// See the last release version : https://github.com/monicahq/monica/releases (15 Feb 2021 update : 2.19.1 at the moment)
```

Comment:
This cannot stay here.
Maybe your intention was to add it as a comment? There is no such thing in markdown...

Reply:
I'm removing it.
This was to trigger the discussion but - yeah - this has no place here.